data/projects/enhancing_coala_quickstart.md

```diff
@@ -0,0 +1,37 @@
+coala-quickstart is a tool that helps users to quickly get started with coala. It generates a `.coafile` (coala's configuration file) that is tailored to the project. Currently, coala-quickstart automatically identifies relevant bears for the project based on the languages used and has a clean and simple command-line interface for taking user’s input when required.
+
+However, there’s a scope for improvement in the following areas:
+
+- Utilizing information from common configuration files: Most of the projects have configuration files like `Gruntfile.js`, `Gulpfile.js`, `rakefile`, `package.json`, etc and they may contain some settings for executables that `coala-bears` supports. For example, if any project's `Gulpfile.js` has include all files except `somefile.css` for `csslint`, it would be nice if our generated `.coafile` is able to include `CSSLintBear` and reflects this change as well.
+- Improving the interface: 
+	- Fixing warnings: Currently, there are lots of warnings which is not user-friendly.
```

Comment:
We don't need to preface every statement with a `Currently`. It'd be much better to make everything a single sentence without the part before the colon. Like,

>Fix non-user-friendly warnings
>Validate user input before writing to the generated coafile
>Add CI integration for Windows (Appveyor) and OS X (Travis)